configure.ac

```diff
@@ -37,7 +37,7 @@ dnl Checks for libraries.
 
 dnl Checks for header files.
 AC_CHECK_HEADERS(crypt.h utmp.h \
-	termio.h sgtty.h sys/ioctl.h syslog.h paths.h \
```

Comment:
I suppose we could also remove the code that depends on the `USE_SYSLOG` macro, but I have not checked it thoroughly.

Reply:
I don't know.  I saw this:

```c
$ grep -rn -B5 undef.USE_SYSLOG
src/sulogin.c-54-/*
src/sulogin.c-55- * syslogd is usually not running at the time when sulogin is typically
src/sulogin.c-56- * called, cluttering the screen with unnecessary messages. Suggested by
src/sulogin.c-57- * Ivan Nejgebauer <ian@unsux.ns.ac.yu>.  --marekm
src/sulogin.c-58- */
src/sulogin.c:59:#undef USE_SYSLOG
```

Reply:
ufff it's kind of difficult to understand. So, on the one hand USE_SYSLOG is defined, but on the other `sulogin.c` undefines it and there's several places where USE_SYSLOG is checked...

Reply:
@hallyn Do you know anything about it?

Reply:
> @hallyn Do you know anything about it?

No - but I do see in NEWS:
1938 - started replacing lots of #ifdef USE_SYSLOG with cleaner macros

I can't tell from the github view here what hunk this was in reference to, but if you're proposing dropping USE_SYSLOG altogether that sounds good.

Reply:
Yes, I'd like to eliminate the use of USE_SYSLOG, but I think it can be done in future PRs.